lib/BC/SleighLifter.cpp

```diff
@@ -1263,6 +1266,22 @@ class SleighLifter::PcodeToLLVMEmitIntoBlock {
                                                   vars[2]);
         return kLiftedInstruction;
       }
+      if (other_func_name == kSysCallName &&
+          insn.arch_name == ArchName::kArchPPC) {
+        DLOG(INFO) << "Invoking syscall";
+        const auto [mem_ptr_ref, mem_ptr_ref_type] =
+            insn_lifter_parent.LoadRegAddress(
+                bldr.GetInsertBlock(), state_pointer, kMemoryVariableName);
+
+        llvm::Value *args[] = {
+            state_pointer, mem_ptr_ref,
+            llvm::ConstantInt::get(
```

Comment:
Extract out - look at the indents, hard to follow.

Reply:
[51a495e](https://github.com/lifting-bits/remill/pull/653/commits/51a495ec193a2724467694a3b0d58d1274a50afa)